Source/common/SNTConfigurator.m

```diff
@@ -715,6 +717,20 @@ - (NSString *)syncClientAuthCertificateIssuer {
   return self.configState[kClientAuthCertificateIssuerKey];
 }
 
+- (SNTSyncCompressionEncoding)syncClientContentEncoding {
+  NSString *contentEncoding = [self.configState[kClientContentEncoding] lowercaseString];
+  if ([contentEncoding isEqualToString:@"zlib"]) {
```

Comment:
Make this deflate, to match what the sent heading is

Reply:
I find this a bit confusing. It seems odd to have the configuration string of `gzip` or `deflate` be either a format or algorithm.

Reply:
We talked offline a bit. I guess it's odd, but fairly consistent with how inconsistently these things are referred to throughout the web...